validator/client/metrics.go

```diff
@@ -85,6 +85,61 @@ var (
 			"pubkey",
 		},
 	)
+	// ValidatorInclusionDistancesGaugeVec used to keep track of validator inclusion distances by public key.
+	ValidatorInclusionDistancesGaugeVec = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "validator",
+			Name:      "inclusion_distance",
+			Help:      "Inclusion distance of last attestation.",
+		},
+		[]string{
+			"pubkey",
+		},
+	)
+	// ValidatorAttestedSlotsGaugeVec used to keep track of validator attested slots by public key.
+	ValidatorAttestedSlotsGaugeVec = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "validator",
+			Name:      "last_attested_slot",
+			Help:      "Last attested slot.",
+		},
+		[]string{
+			"pubkey",
+		},
+	)
+	// ValidatorCorrectlyVotedSourceGaugeVec used to keep track of validator's accuracy on voting source by public key.
+	ValidatorCorrectlyVotedSourceGaugeVec = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "validator",
+			Name:      "correctly_voted_source",
+			Help:      "True if correctly voted source in last attestation.",
+		},
+		[]string{
+			"pubkey",
+		},
+	)
+	// ValidatorCorrectlyVotedTargetGaugeVec used to keep track of validator's accuracy on voting target by public key.
+	ValidatorCorrectlyVotedTargetGaugeVec = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "validator",
+			Name:      "correctly_voted_target",
+			Help:      "True if correctly voted target in last attestation.",
+		},
+		[]string{
+			"pubkey",
+		},
+	)
+	// ValidatorCorrectlyVotedHeadGaugeVec used to keep track of validator's accuracy on voting target by public key.
```

Comment:
```suggestion
	// ValidatorCorrectlyVotedHeadGaugeVec used to keep track of validator's accuracy on voting head by public key.
```

Reply:
Ahh thanks for catching this. As for the gauge @prestonvanloon is worried about the same. Gauge is needed to get a time series which you need if you want to collect the table above. Getting a histogram is only useful to get averages. This as gauge replicates the table in beaconcha.in. The prometheus dbase can then later be pruned can't it?